cmd/recording.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+  	"encoding/json"
```

Comment:
You can't see it in this diff, but the indentation here mixes spaces and tabs.

Please run `go fmt` on this file.